data/dushmanta05.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Dushmanta",
+  "type": "personal",
+  "bio": "Passionate web developer | Open-source enthusiast | Love football",
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/dushmanta05",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/dushmanta05",
+      "icon": "FaGitHub"
```

Comment:
You may want to replace this with **' FaGithub '**, as this is an invalid icon. :+1: 
```Suggestion
      "icon": "FaGithub"
```

Reply:
Thank you for correcting :)